instance/models/openedx_instance.py

```diff
@@ -111,10 +115,7 @@ def domain(self):
 
         Returns external domain if present, otherwise falls back to internal domain.
         """
-        if self.external_lms_domain:
-            return self.external_lms_domain
-        else:
-            return self.internal_lms_domain
+        return self.get_domain('lms')
```

Comment:
@smarnach - sorry, are you asking me "why include this code" or asking @e-kolpakov "why move them to a constant"?

My reasoning: We're adding a couple more domains that can be either internal or external, and I wanted to DRY up the specific implementation of doing that.

An alternative: move to a mixin with a `__getattr__` magic method, and a blessed list of names such that if accessing the attribute `blessed_domain`, it does the `external_blessed_domain`/`internal_blessed_domain` fetching automatically.